GOVERNANCE.md

```diff
@@ -0,0 +1,149 @@
+direction of Rez, as defined in the project
+[charter](ASWF/charter/Rez-Technical-Charter.md).  This
+charter defines the TSC member terms and succession policies.
+
+TODO: Is there a rez charter? I don't know whether it belongs here or in a
+charter, but there should be some sort of mission statement that includes
+goals of increasing adoption of rez and growing the ecosystem of tools and the community of developers around rez.
+
+The responsibilities of the TSC include:
+
+* Coordinating technical direction of the project.
+
+* Project governance and contribution policy.
+
+* GitHub repository administration.
+
+* Maintaining the list of additional Committers.
+
+* Appointing representatives to work with other open source or open
+  standards communities.
+
+* Discussions, seeking consensus, and where necessary, voting on
+  technical matters relating to the code base that affect multiple
+  projects.
+
+* Coordinating any marketing, events, or communications regarding the
+  project.
+
+The TSC elects a Chair person, who acts as the project manager,
+organizing meetings and providing oversight to project
+administration. The Chair is elected by the TSC.  The Chair also
+serves as the Rez representative on the Academy Software
+Foundation (ASWF) Technical Advisory Council (TAC). The chair
+represents the project at ASWF TAC meetings.
+
+### Current TSC Members
+
+* Allan Johns (chair, author) - Nvidia
+* Jean-Christophe Morin
+* Stephen Mackenzie
+* Thorsten Kaufmann
+* Brendan Abel - Walt Disney Imagineering
+* John Mertic - The Linux Foundation (TODO: is John a TSC member?)
+
+### TSC Meetings
+
+All meetings of the TSC are open to participation by any member of the
+Rez community. Meeting times are listed in the [ASWF technical
+community calendar](https://lists.aswf.io/g/tac/calendar). The TSC
+Chair moderates the meeting, or appoints another TSC member to
+moderate in his or her absence.
+
+Items are added to the TSC agenda which are considered contentious or
+are modifications of governance, contribution policy, TSC membership,
+or release process, in addition to topics involving the high-level
+technical direction of the project.
+
+The intention of the agenda is not to approve or review all
+patches. That should happen continuously on GitHub and be handled by
+the larger group of Committers.
+
+Any community member or Contributor can ask that something be reviewed
+by the TSC at the meeting by logging a GitHub issue. Any Committer,
+TSC member, or the meeting chair can bring the issue to the TSC's
+attention by applying the `TSC` label.
+
+Prior to each TSC meeting, the meeting chair will share the agenda with members
+of the TSC. TSC members can also add items to the agenda at the beginning of
+each meeting. The meeting chair and the TSC cannot veto or remove items.
+
+The TSC may invite additional persons to participate in a non-voting capacity.
+
+The meeting chair is responsible for archiving the minutes, stored at 
```

Comment:
This could be changed to secretary, and the pointer to where they are stored can be changed as well. https://github.com/AcademySoftwareFoundation/rez/tree/master/ASWF/TSC/meeting-notes